docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 10
+title: Getting Started
+description: "You're 3 steps away from adding great in-app support to your React native application."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Getting Started {#getting-started}
+
+<Intro>
+
+You're 3 steps away from adding great in-app support to your React Native application.
+
+</Intro>
+
+Guide to integrating the React Native plugin for the Helpshift SDK X.
+
+## Requirements {#version-requirements}
+
+- Node v16 or above
+- Android Studio
+- Xcode 13 or above
+- Java 11 or above
+
+## Download Helpshift React native SDK {#getting-sdk}
+
+#### SDK X - In-app Customer Service {#standard-sdk}
+
+<DownloadButton
+  text="Download Plugin"
+  link="[% sdkx-react-native.plugin_url %]"
+  classes="mb-4"
+/>
+
+
+## Add Helpshift React native plugin to your React Native project {#adding-to-project}
+
+- Add helpshift react native plugin dependancy in your package.json file
+- npm i helpshift-plugin-sdkx-react-native
+- After that, re-run you react application and start using helpshift sdkx functions
```

Comment:
...re-run your application... SDK X, not sdkx

Reply:
Updated.